content/en/dashboards/reporting.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Reporting
+kind: documentation
+disable_toc: false
+is_public: true
+beta: true
+is_beta: true
+further_reading:
+  - link: "https://www.datadoghq.com/blog/template-variable-associated-values/"
+    tag: "Blog"
+    text: "Use associated template variables to refine your dashboards"
+  - link: "https://learn.datadoghq.com/course/view.php?id=8"
+    tag: "Learning Center"
+    text: "Building Better Dashboards"
+---
+
+<div class="alert alert-warning">This feature is currently in private beta. To request access, reach out to support@datadoghq.com.</div>
+
+## Overview
+
+Reporting automatically sends a visual summary of a dashboard to selected recipients on a schedule. The generated reports display images of widgets from a dashboard in a linear HTML format.
+
+{{< img src="dashboards/reporting/report_example.png" alt="An example report titled Checkout KPI Report showing a date, a description, a dashboard link, and 3 images of dashboard widgets" style="width:100%;" >}}
```

Comment:
The `+/-` icons on the Geomap bother me; on the one hand it's a faithful representation of reports, on the other hand it looks wrong and seeing it here makes me wish we had fixed this.

I'm going to see if I can take these icons out of the Geomap snapshots; if I can get this working I'll let you know & ask for a revised snapshot.

Reply:
Took a look at this yesterday, was able to make a PR to fix it: https://github.com/DataDog/web-ui/pull/61225, currently in review.